src/app/home/home.component.ts

```diff
@@ -0,0 +1,13 @@
+import { Component } from '@angular/core';
+
+@Component({
+    selector: 'sh-home',
+    templateUrl: './home.template.html',
+    styleUrls: [ './home.style.scss' ]
+})
+export class HomeComponent {
+
+    public onCreateViewClick(): void {
```

Comment:
If you create this for future, will be better to leave comment about this, because it looks like not finished solution.